interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: `This` refers to an object calling upon itself. Developers can call upon values within an object while inside that same object. 
 
-  Researched answer:
+  Researched answer: `This` is a way to refer to the object when you are inside of said object. It can also be used alone (refers to a global object in this case. )
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a js library that is used to develop the UI for end users
 
-  Researched answer:
+  Researched answer: React (aka React.js) is a Javascript library that is used to build user interfaces by using components. It was developed by facebook in 2013. It can be used to build webpages, apps, games, and more.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: The constructor() is required in React
 
-  Researched answer:
+  Researched answer: After looking it up, the only required lifecycle method required is the render() method. Constructor(), according to the React website, does NOT need to be implemented in all cases. An example of this case would be if you don't bind any methods in your app. 
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX = JS + HTML specifically designed for React. I believe the difference is that JSX allows developers to write in both languages at the same time. 
 
-  Researched answer:
+  Researched answer: One notable difference between HTML and JSX is that JSX demands the usage of fragments `<> </>` in order to compile. Also, all tags are self-closing tags in JSX; you can just write something like <br /> instead of <br></br>. Lastly, all attributes are written in camelCase for JSX.
```

Comment:
I can see how it would be thought that everything is self enclosing. The html tags that we used in the examples were self enclosing  like the break tag `<br />`. However, the h tags like h1, h2 tags are not, `<h1> </h1>, <h2> </h2>`. Please look over the markup language tags. 
These are for quick references only. https://htmlcheatsheet.com/  or http://www.simplehtmlguide.com/cheatsheet.php

Reply:
You are correct. I corrected myself; the React components use the self-closing tags.